packages/core/include/combo/oot/actors/Custom_Torch2.h

```diff
@@ -0,0 +1,32 @@
+#ifndef Z_EN_TORCH2_H
+#define Z_EN_TORCH2_H
+
+#include <combo/common/actor.h>
+
+struct EnTorch2;
+
+typedef struct EnTorch2 {
+    /* 0x000 */ Actor base;
```

Comment:
Style: No offset comment, aligned types

Reply:
I dont understand what you mean by aligned types.